generate/bin/src/Run.purs

```diff
@@ -434,7 +434,7 @@ writePursTidyUpdates updates = do
       pure (updateStable (updateUnstable named))
 
   case named' of
-    Nothing -> Console.log "Failed to update named manifest" *> liftEffect (Process.exit 1)
+    Nothing -> Console.log "Failed to update named manifest" *> liftEffect (Process.exit' 1)
```

Comment:
IDK. @thomashoneyman?

BTW, there are quite many

```purs
Console.log "something"
liftEffect $ Process.exit' 1
```

It may be better to use a dedicated function.

Reply:
Sure, it probably ought to be `Console.error`; the way this is run all the output is going to the console so there won't be an observable difference, but it's the proper thing to do.

@deemp we could add a `die` function that wraps this up, I'm happy with that. It's what Spago does.

Reply:
applied the suggestion in https://github.com/thomashoneyman/purescript-overlay/pull/71/commits/62ce26e10049f535f9acda09ca68332df888ea77